|SOP #WEO-0011                      |<p>v1.0</p><p>Orr, Walker (NIH/NIAID) [F]</p><p>9/13/23 7:18:00 PM</p>|
| :- | -: |

**Sequencing Library Preparation**

*Walker Orr, 08/15/202*


**Initial Library PCR**

|**Steps**|
| :- |
|<p>1.1. Perform PCR on the region of interest. For EV-A71, the primers are as follows:</p><p>&emsp;a. Capsid: WB103/WB126</p><p>&emsp;b. Repl. Prot.: WB283/WB284</p><p>Set up 4 identical 50 ul reactions for each PCR using Q5 DNA Polymerase.</p><p>- 25 ng template DNA per reaction</p><p></p><p>PCR Condition:</p>|

||Initial Denature|98C|1 min.|
| :- | :- | :- | :- |
|25x|Denature|98C|10 sec.|
||Anneal|65C|30 sec.|
||Extend|72C|varies|
||Final Extension|72C|varies|

|<p></p><p>- For the capsid region, use 3 mins. Extension per cycle and 4 mins. Final extension.</p><p>- For the replication proteins, use 4 mins. Extension per cycle and 5 mins. Final extension.</p><p></p>|
| :- |
|<p>1.2. Gel purify and sequence PCR reactions.</p><p>- Pool all capsid PCR reactions; pool all replication protein PCR reactions.</p><p>- Mix 50 ul of each reaction pool with 10 ul 6x loading dye.</p><p>- Run the gel and purify; elute in 8 ul nuclease free water.</p><p>- Store cleaned up DNA at -30C until use.</p>|



**Library Preparation with Twist Universal Adapter System**

*Steps adapted from Twist Publication Doc-001239 Rev. 0.5: “Library Preparation EF 2.0 with Enzymatic Fragmentation and Twist Universal Adapter System.”*

|**DNA Fragmentation, End Repair, and dA-Tailing**|
| :- |
|<p>2.1.1. Prepare the following reagents at room temperature.</p><p>&emsp;a. Qubit BR Assay mix (1 for each sample)</p><p>2.1.2. Thaw the following reagents and keep on ice:</p><p>&emsp;a. Nuclease-free water</p><p>&emsp;b. 75 ng DNA per sample</p><p>&emsp;c. Frag/AT Buffer (Twist Kit)</p><p>&emsp;d. Frag/AT Enzymes (Twist Kit)</p>|
|2.1.3. Program a thermal cycler with the following conditions:|

|Step|Temperature|Time|
| :- | :- | :- |
|1|4C|Hold|
|2|37C\*|20 min.\*|
|3|65C|30 min.|
|4|4C|Hold|

|<p>\*This is expected to give 180-220 bp fragments, for other options, consult the Twist manual.</p><p>Start the program to pre-chill the thermocycler; lid temp should be 105C.</p>|
| :- |
|2.1.4. Mix gDNA by flicking the tube. Quantitate with Qubit BR to determine concentration.|
|2.1.5. Dilute DNA to a final concentration of 1.25 ng/ul in chilled nuclease-free water. To have sufficient DNA with overage, use a total volume of 60 ul water (75 ng total DNA)|
|2.1.6. Add 40 ul of each diluted sample (50 ng total DNA) into a PCR tube. Pipette up and down and spin down.|
|**DNA Fragmentation, End Repair, and dA-Tailing**|
|2.2.1. Vortex Frag/AT Buffer for 5 seconds. Spin down. Invert Frag/AT Enzymes >10 times. Spin down.|
|2.2.2. Prepare a Frag/AT Master Mix. **Per reaction**: **4 ul Buffer and 6 ul Enzymes.** Pipette up and down 10 times without introducing air bubbles.|
|2.2.3. Add **10 ul Master Mix** to each **40 ul DNA** sample tube. Pipette up and down >10 times without introducing air bubbles and spin down. Cap tubes and keep on ice.|
|2.2.4. Spin down and immediately transfer to the pre-chilled thermocycler. Initiate the remaining steps of the thermocycling program. Begin preparing reagents for the next step.|
|**Ligate Twist Universal Adapters and Purify**|
|<p>2.3.1. Thaw the following reagents and keep on ice:</p><p>&emsp;a. Twist Universal Adapters</p><p>&emsp;b. Ligation Master Mix.</p><p>2.3.2. Prepare fresh 80% ethanol (1 mL for each sample): Add 250 ul Nuclease-free water to 1000 ul 100% Ethanol, for each sample.</p><p>2.3.3. Allow DNA purification beads to stand at room temperature for at least 30 minutes before use. These are stored at 4C.</p><p>2.3.4. Program a thermal cycler to incubate samples at 20C with the headed lid turned off. Start the program so that the cycler has reached 20C when the samples are done being prepared.</p>|
|2.3.5. Add **5 ul** **Twist Universal Adapters** into each sample well or tube containing the dA-tailed DNA from Step 3. Mix by gently pipetting up and keep on ice.|
|2.3.6. Invert **Ligation Master Mix** a minimum of 10 times until homogenized and place on ice. Do not vortex. Add **20 ul** to each sample; pipette up and down and spin down.|
|2.3.7. Incubate for 15 minutes at 20C in the thermal cycler, then move the samples to the benchtop. Proceed to Step 2.3.9 (Bead purification). This is a good time to make 80% EtOH.|
|2.3.8. Vortex room-temperature beads until well mixed.|
|2.3.9. Add **60 ul homogenized beads** to each ligation sample. Transfer to a 1.5-mL DNA low bind tube and mix well by vortexing. Incubate samples for 5 minutes at room temperature. |
|2.3.10. Spin down, then place samples on a magnetic plate for 2 minutes or until supernatant is clear. Carefully drain supernatant with a P200 pipette, touching the tip to the side of the tube opposite the pellet.|
|2.3.11. Add **200 ul fresh 80% ethanol** without disturbing the pellet; dispense liquid on the side of the tube opposite the pellet to do this. Incubate 1 minute, then drain the ethanol. |
|2.3.12. Repeat the wash step in 4.11, for a total of two washes. After the second wash, briefly spin down to collect all ethanol, then return to the plate.|
|2.3.13. Drain as before but use a P20 to aspirate the last little bit of ethanol. Dry for 5 minutes at room temperature. Don’t overdry.|
|2.3.14. Remove tube from magnetic plate and add **17 ul nuclease-free water** to each sample. Mix by pipetting until homogenized. Incubate at room temperature for 2 minutes. Spin down, transfer to the magnetic plate and incubate a further 3 minutes.|
|2.3.15. Transfer 15 ul clear supernatant to a clean 0.2 mL PCR tube without disturbing the pellet.|
|**PCR Amplify Using Twist UDI Primers and Purify**|
|<p>2.4.1. Thaw and keep on ice:</p><p>&emsp;a. Equinox Library Amp Mix (2x)</p><p>&emsp;b. Twist UDI Primers. You may wish to spin the plate down before use.</p>|
|2.4.2. Program a thermal cycler with the following conditions. Set the temperature of the lid to 105C. Reaction volume will be 50 ul.|

||Initial Denature|98C|45 sec.|
| :- | :- | :- | :- |
|6-8x|Denature|98C|15 sec.|
||Anneal|60C|30 sec.|
||Extend|72C|30 sec.|
||Final Extension|72C|1 min.|
||Hold|4C|Until use.|

||
| :- |
|<p>2.4.3. Add to each sample 10 ul of a distinct Twist UDI Primer from the provided 96-well plate and mix well by gentle pipetting.</p><p>- Make sure there are no air bubbles at the bottom the plate and that you indeed aspirated the primer.</p><p>- Note in your lab notebook which Plate (A,B,C,D) and primer (well) was used for each sample!</p>|
|2.4.4. Invert Equinox Library Amp Mix 5 times immediately before use; do not vortex. Add 25 ul of Equinox Library Amp Mix (2x) and mix well by gentle pipetting. Spin down and transfer to thermocycler. Start thermocyling program.|
|2.4.5. Remove samples from the block and proceed to purification.|
|<p>2.4.6. Repeat the bead-based purification used in Steps 2.3.8 thru 2.3.15, with the following deviations:</p><p>- Use 1x beads (50 ul) for each reaction.</p><p>- Resuspend dried beads in 22 ul of water; withdraw 20 ul of purified sample at the end. Transfer samples to 0.2-mL PCR tubes.</p>|
|**QC Checks**|
|<p>2.5.1. Quantify DNA using the Qubit BR Kit. Record this information in your lab notebook.</p><p>2.5.2. Run samples on the TapeStation (Use the D1000 HS).</p><p>- You should expect to see samples in the size range of 350-425 bp.</p><p>- Record average length of DNA corresponding to the peak in your lab notebook; this value will be required for molarity computation and Illumina sequencing.</p><p>2.5.3. Store amplified indexed libraries at -20C until use in the Sequencing reaction.</p>|

**

**Sequencing using Illumina MiSeq**

*Steps adapted from Zhou, S., Hill, C. S., Clark, M. U., Sheahan, T. P., Baric, R. and Swanstrom, R. (2021). Primer ID Next-Generation Sequencing for the Analysis of a Broad Spectrum Antiviral Induced Transition Mutations and Errors Rates in a Coronavirus Genome . Bio-protocol 11(5): e3938. DOI: 10.21769/BioProtoc.3938.*

|**Preparing the Sequencer**|
| :- |
|<p>3.1.1. Wash the sequencer with 0.5% Tween-20 in MilliQ water. Use the “Maintainence Wash” protocol; follow the directions in the manual and from the machine. There are a total of three washes that take around 30 minutes each.</p><p>- Note that a flow cell (from the previous run, for instance) needs to be in the machine for the washes to work.</p>|
|**Preparing the Cartridge and Reagents**|
|3.2.1. Remove the cartridge (Box 1) from -20C storage. Place the cartridge in a water bath (the purple ice buckets are a good size for this) consisting of room temperature DI water to submerge the base of the cartridge. Do not allow the water to exceed the maximum water line printed on the cartridge.|
|3.2.2. Allow the reagent cartridge to thaw for approximately 60 minutes. Also thaw Buffer HT1 on ice.|
|**Preparing the DNA Library**|
|<p>3.3.1. Prepare the following reagents on ice:</p><p>&emsp;&emsp;i. PhiX DNA (stored at -30C)</p><p>&emsp;&emsp;ii. Sample DNA from Twist library prep workflow.</p><p>3.3.2. Also prepare the following reagents at room temperature:</p><p>&emsp;&emsp;i. 0.2 N NaOH. Prepare fresh dilution from 10N stock by adding 20 ul 10N NaOH to 980 ul Nuclease-free water. </p><p>&emsp;&emsp;ii. Find and keep handy: 10 mM Tris-HCl, pH 8.5 with 0.1% Tween-20 (8 ul/sample)</p>|
|<p>3.3.3. Calculate the molarity of the pooled DNA based on the Qubit concentration and the average DNA size and dilute to 4 nM. Keep diluted DNA on ice.</p><p>- Use this formula to determine the molarity of your DNA: </p><p>concentration in ng/ulaverage library size in bp\*106660gmol=concentration in nM</p>|
|3.3.4. Denature the DNA by adding 5 ul 4nM DNA to 5 ul 0.2N NaOH. Vortex briefly, spin down, and incubate for 5 minutes at room temperature.|
|3.3.5. Add 10 ul (all of the sample from the above step) to 990 ul pre-chilled HT1. Mix by pipetting.|
|3.3.6. Perform final dilution: add 400 ul of the above dilution to 600 ul of pre-chilled HT1. Vortex briefly and spin down. Keep on ice.|
|3.3.7. Prepare PhiX control DNA. Start by adding 2 ul of PhiX to 8 ul 10 mM Tris-HCl, pH 8.5 with 0.1% Tween-20, in a microcentrifuge tube. Then, add 10 ul 0.2N NaOH. Vortex and spin and incubate for 5 minutes at room temperature.|
|3.3.8. Add 980 ul pre-chilled HT1 to the denatured PhiX DNA.|
|3.3.9. Perform a final dilution of the PhiX DNA by adding 400 ul of the above sample to 600 ul of pre-chilled HT1.|
|3.3.10. Prepare sample DNA with 15% PhiX spike-in: Mix 105 ul PhiX library with 595 ul sample library. Pipette up and down, spin down, and keep on ice.|
|**Loading Prepared Sample on the Cartridge**|
|3.4.1. Remove the cartridge from the water bath and gently tap it on the bench to dislodge water from the base of the cartridge. Dry the base of the cartridge with towels, if necessary.|
|3.4.2. Invert the cartridge gently ten times to mix. The motion should be similar to a 3-D shaker: gentle and continuous. Visually inspect that all positions are thawed and free of precipitates, paying special attention to buffer IMT (purple).|
|3.4.3. Gently tap cartridge on the bench to dislodge air bubbles in the reagents.|
|3.4.4. Use a clean, empty P1000 to puncture position 17 (“Load Samples”) on the reagent cartridge.|
|3.4.5. Pipette 700 ul (i.e., all) of the sample library into the Load Samples reservoir.|
|**Starting the MiSeq**|
|3.5.1. Set up a file containing the UDI information from Twist. Use the template Sample Sheet file from Twist; you can edit in Excel but save as a CSV on a portable drive with USBA connection which you will carry to the MiSeq.|
|3.5.2. Find the flow cell and incorporation buffer in Box 2 (stored at 4C). Remove the flow cell from its buffer and follow the instructions in the MiSeq booklet for washing it. You’ll need Isopropanol for this step. Only use lint-free Kim Wipes for cleaning.|
|3.5.3. Insert the flow cell, cartridge, and bottle of incorporation buffer into the MiSeq. Also, plug your portable drive into the MiSeq. Verify that your CSV has the correct information and select it when setting up your run. After finishing the prompts to start the run, you will need to wait with the machine while it goes through the flow rate test; this can take up to 5 minutes or so.|


1

